### PR TITLE
Add basic tvOS SDK target.

### PR DESCRIPTION
**Pull request by nlutsenko**
First step in big game.
Contributes to #250

### CODE REVIEW

**Comment by waterclock** [comment]
Does this compile? I kinda need a tvOS Parse framework that will compile, even if it's wildly broken as I need to submit a provisional build to iTunesConnect to test some StoreKit code. Thanks!

**Comment by nlutsenko** [comment]
Nope, not yet... :)
Will make it compile very very soon.

File: Configurations/Bolts-tvOS.xcconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//
+// Copyright (c) 2015-present, Parse, LLC.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+//
+
+SDKROOT = appletvos
```
Comment (richardjrossiii) [inline Configurations/Bolts-tvOS.xcconfig]: I don't think we want this file.

**Comment by fvimagination** [comment]
Hi, just wondering if you will post something here when Parse will be ok with tvOS.
Thanks!

**Comment by eytanschulman** [comment]
I want to submit my tv app today or tomorrow, and Parse Analytics would be killer - i'd really really like some analytics in my app. Does this work yet?

**Pull request merged**